app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    path_elements = None
+    if "/" in split_command[2]:
+        path_elements = split_command[2].split("/")
+    path = ""
+    for element in path_elements:
+        if "." not in element:
+            path += element + "/"
```

Comment:
please read checklist and improve this part of code

Reply:
done